packages/camera/camera_platform_interface/test/camera_platform_interface_test.dart

```diff
@@ -157,13 +157,19 @@ void main() {
 
       // Act & Assert
       expect(
-        () => cameraPlatform.createCamera(
```

Comment:
Why are you reducing test coverage? Adding a new constructor doesn't mean the old one shouldn't be tested.

Reply:
Restored test coverage for `createCamera`. Added same test for `createCameraWithSettings`.